src/tip/packages.py

```diff
@@ -68,7 +69,7 @@ def _install(package_specifier: str):
     if os.path.exists(package_dir):
         return
     os.makedirs(package_dir)
-    command = f"pip install --target={package_dir} {package_specifier}"
+    command = f"pip install --target={package_dir} --no-deps {package_specifier}"
```

Comment:
It doesn't look elegant. Try to remember absolute path to `pip` (in env var?) then create alias or modify `PATH` so that `tip` will be called instead of `pip` while installing dependencies.